src/sentry/tasks/invite_missing_org_members.py

```diff
@@ -84,18 +89,33 @@ def send_nudge_email(org_id):
         )
         return
 
+    shared_domain = _get_shared_email_domain(organization)
+
+    if shared_domain:
+        commit_author_query = commit_author_query.filter(email__endswith=shared_domain)
+    else:
+        for filtered_email in FILTERED_EMAIL_DOMAINS:
+            commit_author_query = commit_author_query.exclude(email__endswith=filtered_email)
+
+    for filtered_character in FILTERED_CHARACTERS:
+        commit_author_query = commit_author_query.exclude(email__icontains=filtered_character)
+
     commit_authors = []
-    for commit_author in commit_author_query:
+    for commit_author in commit_author_query[:3]:
```

Comment:
just curious, is there a reason why this is sliced to 3?

Reply:
we show the top 3 most active contributors in the email, so we don't need to iterate over anything after that